pkg/process/config/config_test.go

```diff
@@ -410,60 +402,6 @@ func TestProxyEnv(t *testing.T) {
 	os.Unsetenv("PROXY_PASSWORD")
 }
 
-func TestEnvSiteConfig(t *testing.T) {
```

Comment:
I recreated these tests in `cmd/process-agent/config_test.go`, and since these are the only usages of `TestEnvSiteConfig*.yaml` I deleted them